unitOneAssessmentPractice.js

```diff
@@ -3,50 +3,131 @@ let assert = require('assert')
 // Question One:
 
 // Write a function called double that doubles a number
+const double = (num) => {
+  return num * 2
+}
+
 
 // Uncomment out the next line to test your solution
-// runQ1Tests()
+runQ1Tests()
+
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
+const containsSeven = (num) => {
+    let str = num.toString()
+    if(str.includes("7")){
```

Comment:
You can simply return str.includes("7") since includes already returns a boolean